libs/sdk-ui-dashboard/src/_staging/drills/test/InsightDrillDefinitionUtils.test.ts

```diff
@@ -0,0 +1,110 @@
+// (C) 2021 GoodData Corporation
+import { SimpleDashboardSimpleSortedTableWidgetDrillTargets } from "../../../model/tests/Dashboard.fixtures";
+import { IDrillToDashboard } from "@gooddata/sdk-backend-spi";
+import { validateDrillDefinitionOrigin } from "../InsightDrillDefinitionUtils";
+import { uriRef } from "@gooddata/sdk-model";
+import { IAvailableDrillTargets } from "@gooddata/sdk-ui";
+
+describe("InsightDrillDefinitionUtils", () => {
+    describe("validateDrillDefinitionOrigin", () => {
+        const drillTargets: IAvailableDrillTargets = SimpleDashboardSimpleSortedTableWidgetDrillTargets;
+
+        it("should not throw error for valid measure origin drill definition ", async () => {
+            const validMeasureOriginDrillDefinition: IDrillToDashboard = {
+                type: "drillToDashboard",
+                origin: {
+                    type: "drillFromMeasure",
+                    measure: {
+                        localIdentifier: "31c22194386b408aa80ab90b966e85a7",
```

Comment:
We should probably use some constants for these localId and id values. Should they be in the fixtures file?

Reply:
Done